src/components/expanded-state/swap-details/SwapDetailsContent.js

```diff
@@ -112,10 +112,10 @@ export default function SwapDetailsContent({ isHighPriceImpact, isRefuelTx, onCo
             <Rows space="24px">
               <Separator color="divider80 (Deprecated)" />
               <SwapDetailsPriceRow testID="swaps-details-price-row" tradeDetails={tradeDetails} />
-              {!isNativeAsset(inputCurrency?.address, inputCurrencyNetwork) && (
+              {!isNativeAsset(inputCurrency?.address, inputChainId) && (
                 <SwapDetailsContractRow asset={inputCurrency} onCopySwapDetailsText={onCopySwapDetailsText} />
               )}
-              {!isNativeAsset(outputCurrency?.address, inputCurrencyNetwork) && (
+              {!isNativeAsset(outputCurrency?.address, outputChainId) && (
```

Comment:
noticed that this went from `inputCurrencyNetwork` to `outputChainId` and wanted to check if it was intentional

but also, this file may no longer be used anyway (if this is part of old swap flow)

Reply:
i changed it on purpose, this is for output currency but it was using inputCurrencyNetwork, same as above